main.cpp

```diff
@@ -13,6 +13,12 @@
 constexpr auto MARGINCONFIGPATH =
     "/usr/share/read-margin-temp/config-margin.json";
 
+constexpr auto debugEnablePath = "/etc/thermal.d/margindebug";
+extern bool debugEnabled;
+
+constexpr auto spofDisablePath = "/etc/thermal.d/spofdisable";
```

Comment:
Hi Krellan, as far as I understand, if I use the command parsing "--strick", this file will not need, is it?

Reply:
That's something that should be merged together. There should be 2 command line options:

--debug = sets debugEnabled to true (default false)

--strict = sets spofEnabled to true (default false)

My usage of spofEnabled should be similar to your usage of the --strict option. I think they can be merged into a single option that controls both, as they are both part of a complete solution (I am checking for "NaN" values and file I/O errors, and you are checking the "Operational" interface, so it makes sense that these would stack together).

Alternatively, have --spof and --strict remain separate command-line options.

Reply:
Oh I see, I will todo this. By the way, I think it can check "Operational", "Warning", "Ctritical" three interface not only "Operational" one interface(something like phosphor-pid-control set failsafe mode method), and I'm implement this. How do you think?

Reply:
Sounds good to me. As for the "Operational" interface, I do not currently use this interface, so I don't have any feedback as to how it should be used.